bin/build-ember

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+
+rm -rf build/ember-app
+cd src/ember-app
+./node_modules/ember-cli/bin/ember build --output-path ../../build/ember-app/
```

Comment:
When I do production mode, the build files now have hashes in them. Is there any way I can configure ember cli to not change the build file name every time I run the build? Otherwise I'm not sure how to `<script>` tag the vendor and app bundles.

<img width="465" alt="screen shot 2017-08-04 at 10 11 58 am" src="https://user-images.githubusercontent.com/5524384/28977101-eecd49b8-78fd-11e7-87b4-7e3d331fb63e.png">